### PR TITLE
supply quickfix to remove unused macro

### PR DESCRIPTION
**Pull request by fangzhzh**
### Description
supply quickfix to remove unused macro

Fixes erlang-ls#882

### CODE REVIEW

File: apps/els_lsp/test/els_code_action_SUITE.erl [hunk 1]
```diff
@@ -53,14 +54,18 @@ init_per_testcase(TestCase, Config) ->
 -spec end_per_testcase(atom(), config()) -> ok.
 end_per_testcase(TestCase, Config) ->
   els_test_utils:end_per_testcase(TestCase, Config).
+%%==============================================================================
+%% Const
+%%==============================================================================
+-define(COMMENTS_LINES, 2).
```
Comment (robertoaloi) [inline apps/els_lsp/test/els_code_action_SUITE.erl:60]: Clever. This offset-based approach is something we could use in many other test suites...

File: apps/els_lsp/test/els_code_action_SUITE.erl [hunk 2]
```diff
@@ -156,3 +161,28 @@ fix_module_name(Config) ->
     ],
   ?assertEqual(Expected, Result),
   ok.
+
+-spec remove_unused_macro(config()) -> ok.
+remove_unused_macro(Config) ->
+  Uri = ?config(code_action_uri, Config),
+  Range = els_protocol:range(#{from => {?COMMENTS_LINES+17, 9}, to => {?COMMENTS_LINES+17, 15}}),
```
Comment (robertoaloi) [inline apps/els_lsp/test/els_code_action_SUITE.erl:168]: We will probably get a lint error here (spaces around +) in CI.

**Review by robertoaloi — APPROVED** [review]
Amazing work! I love how you factored out the "current line" function. Thanks for your contribution!

File: apps/els_lsp/test/els_code_action_SUITE.erl (hunk 2 above)
Comment (fangzhzh) [reply apps/els_lsp/test/els_code_action_SUITE.erl]: addressed in a3d3b9a

**Review by robertoaloi — APPROVED** [review]

**Comment by robertoaloi** [comment]
Thanks!

**Pull request merged**